apps/frontend/src/components/proposal/ProposalTableInstrumentScientist.tsx

```diff
@@ -257,7 +257,7 @@ const ProposalTableInstrumentScientist = ({
 }: {
   confirm: WithConfirmType;
 }) => {
-  const [currentPage, setCurrentPage] = useState(0);
+  const currentPage = 0;
```

Comment:
@bashanlam I have a suggestion. Could you check if it make sense. Since this variable doesn't have any dependency on the component now, can we just move it outside the component like this?

`const CURRENT_PAGE = 0` OR `const STARTING_PAGE = 0`. 

It doesn't have to be in the useEffect, as the value is not going to change at all.

Reply:
I've moved the currentPage variable outside the component while retaining its name. However, the page crashes when currentPage is undefined, suggesting it might be a mandatory prop for MaterialTable.